assert/assertions.go

```diff
@@ -1703,7 +1689,6 @@ func NotRegexp(t TestingT, rx interface{}, str interface{}, msgAndArgs ...interf
 	}
 
 	return !match
-
```

Comment:
These files might need formating for sure, but I would encourage you to revert all these spaces changes. It mess up the history of the files.

If you want to format them, why not, but then please open a PR about it or use a separate commit in your current PR

Reply:
makes sense, undid the auto formatting: https://github.com/stretchr/testify/pull/1690/commits/bdff01450338f98f2c043d0c08dda4de1ddc9fda